fastai/text/data.py

```diff
@@ -351,7 +351,7 @@ def from_folder(cls, path:PathOrStr, tokenizer:Tokenizer=None, train:str='train'
         "Create a `TextDataBunch` from text files in folders."
         tokenizer = ifnone(tokenizer, Tokenizer())
         path=Path(path)
-        k_names = ['max_vocab', 'chunksize', 'min_freq', 'n_labels', 'clear_cache']
+        k_names = ['max_vocab', 'chunksize', 'min_freq', 'n_labels', 'clear_cache', 'classes']
         txt_kwargs, kwargs = extract_kwargs(k_names, kwargs)
         train_ds = TextDataset.from_folder(path, tokenizer, train, shuffle=shuffle, vocab=vocab, **txt_kwargs)
         datasets = [train_ds, TextDataset.from_folder(path, tokenizer, valid, classes=train_ds.classes,
```

Comment:
The TextDataset.from_folder will fail if 'classes' is specified, because a value for classes will be passed twice: once from train_ds.classes and once from the **txt_kwargs

Reply:
Ah good catch! Do you want to adapt your previous PR with txt_kwargs.pop()?

Reply:
Sure, will do :+1: